modules/core/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "bitgo",
-  "version": "11.9.1",
+  "version": "11.10.1",
```

Comment:
Please do not modify the versions - this is done as part of the release process.

Reply:
Ah sorry, will revert this!